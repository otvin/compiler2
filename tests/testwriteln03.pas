program testwriteln03;
{test passing multiple arguments to write/writeln}
begin {main}
  writeln('hello ', 'world');
  write('I ', 'have ', 4, ' arguments.');
  writeln('!');
end.
