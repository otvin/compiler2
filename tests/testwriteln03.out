hello world
I have 4 arguments.!
